packages/dropdown/src/hooks/useKeyDownHandle.ts

```diff
@@ -1,34 +1,80 @@
+import type { Dispatch, SetStateAction } from 'react';
 import React, { useContext, useCallback, RefObject } from 'react';
 import { DropdownContext } from '../context';
 
 const useDropdownKeyDownHandle = () => {
   const { setOpen } = useContext(DropdownContext);
 
   const handleKeyDown = useCallback(
-    (event: React.KeyboardEvent<HTMLElement>, menuItemRef: RefObject<HTMLLIElement>, onClick: any) => {
+    (
+      event: React.KeyboardEvent<HTMLElement>,
+      menuItemRef: RefObject<HTMLLIElement>,
+      setSubOpen?: Dispatch<SetStateAction<boolean>>
+    ) => {
       if (!menuItemRef.current) return;
 
-      const menuItemEls = menuItemRef.current.parentNode?.querySelectorAll<HTMLElement>('.menuitem');
+      const menuItemEls = menuItemRef.current
+        .closest('.menu')
+        .querySelectorAll<HTMLElement>('.menu_item:not(.disabled)');
+
       if (menuItemEls) {
         const menuItemCount = menuItemEls.length;
-        const currIndex = Array.from(menuItemEls).indexOf(event.currentTarget); // sub1의 currIndex를 못찾음
-        switch (event.key) {
-          case 'ArrowDown':
-          case 'ArrowRight':
-            if (currIndex < menuItemCount! - 1) {
-              event.preventDefault();
-              menuItemEls[currIndex + 1].focus();
+        const currItemIndex = Array.from(menuItemEls).indexOf(event.currentTarget);
+        const currSubItemEls = menuItemEls[currItemIndex]?.querySelectorAll<HTMLElement>('.sub_item:not(.disabled)');
```

Comment:
selector상수는 별도로 관리 되면 좋을것 같아요. 추가로 `disabled`는 html속성이라서 class부여없이 이렇게 선택해도 작동하지 않나용?
```suggestion
        const currSubItemEls = menuItemEls[currItemIndex]?.querySelectorAll<HTMLElement>('.sub_item:not:disabled');
```